core/file.go

```diff
@@ -114,7 +114,8 @@ func CreateFile(prefix string, subfolder string, fileExtension string) (*os.File
 		return nil, err
 	}
 
-	fileName := time.Now().Format("2006-02-01-15-04-05")
+	fileName := time.Now().Format("2006-01-02T15-04-05")
+	fileName = strings.Replace(fileName, "T", "-", 1)
```

Comment:
why `T` and replace with `-`? it works just fine if you use directly `"2006-01-02-15-04-05"`

Reply:
`"2006-01-02-15-04-05"` doesn't work fine, unfortunately - it was the cause of the issue. The `T` is required as a format specifier, otherwise `time.Format()` will think the `01` in `2006-01-02` is the day, not the month. 

I found no simple way of specifying a custom format in `time.Format()`, and the built-in one requires `T`, which is then replaced with a dash.

Reply:
you're right. I've read better the documentation